index.html

```diff
@@ -1,66 +1,110 @@
 ---
+title: openSUSE ­— Open source OS for desktops, servers and containers
+description: openSUSE is an open source operating system based on Linux.
```

Comment:
openSUSE is not an operating system, it's a community that makes some operating systems

Reply:
This also isn't getting translated!

Reply:
I've altered the description. To solve the translation issue, I'd like to switch to doing something like this:

```
title_translation_key: homepage_page_title
description_translation_key: homepage_page_description
```

And this in the translation file:

```
homepage_page_title: Open source desktop and server operating systems
homepage_page_description: openSUSE makes open source operating systems based on Linux.
```

Do you know how I can get Liquid to read out the correct locale variable?

Reply:
Well, you control where those variables go, so you just have to modify the `head`'s strings to use `locale[page.title_translation_key]` or something along these lines

Reply:
Woah, I didn't realize Liquid lets you use an index operator for that. Fixed!